draft-kazuho-httpbis-priority.md

```diff
@@ -168,37 +168,37 @@ priority = urgency=non-blocking, progressive=?1
 
 # Merging Client- and Server-Driven Parameters {#merging}
 
-It is not always the case that the client has the best view of how the HTTP
-responses should be prioritized.  For example, whether a JPEG image should be
-served progressively by the server depends on the structure of that image file
-- a property only known to the server.
+It is not always the case that the client has the best understanding of how the
+HTTP responses deserve to be prioritized.  For example, use of an HTML document
+might depend heavily on one of the inline images.  Existence of such dependency
+is typically best known to the person that administers the document.
 
-Therefore, a server is permitted to send a "Priority" response header field.
-When used, the parameters found in this response header field override those
-specified by the client.
+By using the "Priority" response header, a server can override the
+prioritization hints provided by the client.  When used, the parameters found
+in the response header field overrides those specified by the client.
 
 For example, when the client sends an HTTP request with
 
 ~~~ example
 :method = GET
 :scheme = https
 :authority = example.net
-:path = /image.jpg
+:path = /menu.png
 priority = urgency=non-blocking, progressive=?1
 ~~~
 
 and the origin responds with
 
 ~~~ example
 :status = 200
-content-type = image/jpeg
-priority = progressive=?0
+content-type = image/png
+priority = urgency=document
 ~~~
 
-the intermediary's view of the progressiveness of the response becomes negative,
-because the server-provided value overrides that provided by the client.  The
-urgency is deemed as `non-blocking`, because the server did not specify the
-parameter.
+the intermediary's view of the urgency of the response is promoted from
+`non-blocking` to `document`, because the server-provided value overrides that
+provided by the client.  The progressiveness of the response continues to be
+deemed as true, because the server did not specify the `progressive` parameter.
```

Comment:
This confuses me as it seems to conflict with previous guidance of

```
This document defines the urgency and progressive parameters. Values of these parameters MUST always be present. When any of the defined parameters are omitted, or if the Priority header field is not used, their default values SHOULD be applied.
```

and

```
The default value of the progressive parameter is 0.
```

Are you suggesting a third case where omission in responses means the request value is respected?

Reply:
> Are you suggesting a third case where omission in responses means the request value is respected?

Yes. I think I've better clarified that in 9f30235.

Reply:
I think the new text clarifies this example but I'm not sure that a reader of the parameter definition spec would come to the conclusion that this is the expected behaviour. In my mind, the rules for priority header were quite symmetric but I think this highlights an asymmetry between how they are applied across requests and responses. I think we might want to spin that out into a separate issue, WDYT?

Reply:
Opening a separate issue makes sense to me.